src/vivarium_public_health/disease/special_disease.py

```diff
@@ -11,6 +11,7 @@
 import re
 
 import pandas as pd
+from vivarium.framework.values import list_combiner, joint_value_post_processor
```

Comment:
In this file, and in state.py, I was having a hard time parsing through the data/dependency structures.  They are now reorganized and essentially the same blocks of code (indicating our abstractions are not so great; no surprise)

Reply:
This is all around loading data to set up look up tables and pipelines.

Note, none of the test code needed to change in order to make the test pass.

Reply:
Generically, the pattern is
```
load data (checking conditions as necessary, shove this off to a method that takes the builder as an arg when messy)
turn data into lookup table
use lookup table as source/modifier for appropriate pipelines
``

Reply:
Structurally, riskattributable diseases will now always have mortality data/pipelines, they may just be sourced with all-zero data if appropriate.